corehq/apps/hqadmin/tasks.py

```diff
@@ -180,3 +181,26 @@ def track_es_doc_counts():
                     }
                     metrics_gauge('elasticsearch.shards.docs.count', i['docs']['count'], tags)
                     metrics_gauge('elasticsearch.shards.docs.deleted', i['docs']['deleted'], tags)
+
+
+@periodic_task(queue='background_queue', run_every=crontab(minute="0", hour="0"))
+def track_pg_limits():
+    for db in settings.DATABASES:
+        with connections[db].cursor() as cursor:
+            cursor.execute("select table_name, data_type from information_schema.columns where column_name = 'id'")
```

Comment:
this doesn't seem to be used at all.

Reply:
ah yep. that was from a previous version. fixed.